src/frequenz/sdk/data_ingestion/resampling/subscriptions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Common model definitions used for resampling purposes.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+from frequenz.channels import Receiver, Sender
+
+from frequenz.sdk.core.timeseries import Sample, TimeSeriesId
+
+
+@dataclass(frozen=True)
+class Subscription:
+    """Subscription for receiving data for a specific time series id."""
+
+    time_series_id: TimeSeriesId
+    start_time: Optional[datetime] = None
```

Comment:
Please describe what it means that start_time is None in code
>  start_time: Optional[datetime] = None

Reply:
I assumed that this would change to however it is implement by @sahas-subramanian-frequenz in the data sourcing actor, but had to create some class for it to continue with the rest of the code.

Let me know once it is clear what is should be and I'll update the code.